packages/nimble-components/src/anchor-tab/template.ts

```diff
@@ -0,0 +1,30 @@
+import { html, ViewTemplate } from '@microsoft/fast-element';
+import {
+    endSlotTemplate,
+    FoundationElementTemplate,
+    startSlotTemplate
+} from '@microsoft/fast-foundation';
+import type { AnchorTab, TabOptions } from '.';
+
+export const template: FoundationElementTemplate<
+ViewTemplate<AnchorTab>,
+TabOptions
+> = (context, definition) => html<AnchorTab>`
+    <template slot="anchortab" role="tab" aria-disabled="${x => x.disabled}">
+        ${startSlotTemplate(context, definition)}
+        <a
```

Comment:
The anchor button and anchor have tests that verify that they reflect attributes like this but I don't see similar tests for the anchor tab. Could you add them and also see if there are additional tests we should bring over from the other anchor types?

Reply:
I added tests for the eight link-related attributes. I noticed that we were missing one ('href') in the anchor tests, so I added that as well. There weren't any other link-related tests that needed to be brought over.